kettle/src/main/resources/standalone/configuration/hawkular-realm.json

```diff
@@ -57,17 +57,19 @@
       "secret" : "${uuid.hawkular.accounts.backend}"
     },
     {
-      "name": "hawkular-accounts-ui",
+      "name": "hawkular-ui",
       "enabled": true,
       "bearerOnly" : false,
       "publicClient": true,
-      "secret": "${uuid.hawkular.accounts.ui}",
+      "secret": "${uuid.hawkular.ui}",
       "fullScopeAllowed" : false,
       "redirectUris": [
+        "http://localhost:8080/*",
```

Comment:
I believe the PR https://github.com/hawkular/hawkular/pull/46 address most of the common cases, including the scenario where Keycloak is on the same ip/port as Hawkular. Any other situation is, as I see it, a special (production?) deployment, and a fine tune on this will be required anyway. This fine tune can happen either directly on the embedded Keycloak via the admin UI, or by using an external Keycloak (which we'll have no control of).